### DISCUSSION TITLE
How to get CSS theme variables to work?

### DISCUSSION THREAD

**Discussion by neelduttahere** [xyflow/xyflow#4587] [Q&A]
Based on this section in the documentation: https://reactflow.dev/learn/customization/theming#css-variables , 
How do we make use of this? I am assuming when we import `@import '@xyflow/react/dist/style.css'`, it comes along with the theme variables in the root, but when inspecting in the browser, I don't find them being used anywhere. So naturally when I try to override them, nothing happens. What am I missing?

**Comment by bcakmakoglu** [comment]
All of the CSS variables are bound to the `.react-flow` class name element (the main wrapper of ReactFlow)

<img width="498" alt="image" src="https://github.com/user-attachments/assets/0002c564-27c2-4a50-b52b-a5f85409c223">

[If you want to change a variable value you can do something like this](https://codesandbox.io/p/sandbox/pedantic-aryabhata-9clryv?file=%2Fsrc%2Fstyles.css%3A15%2C34)

```css
.react-flow {
  --xy-edge-stroke-default: pink;
}
```
